### ISSUE TITLE
Loss and 0% Success Rate on Libero-10 with lerobot/xvla-libero Checkpoint

### ISSUE THREAD

**Issue by Hoar012** [huggingface/lerobot#3401]
### Ticket Type

❓ Technical Question

### Environment & System Info

```Shell

```

### Description

Hi,

I ran the example code and obtained a loss of 87.50. I'm wondering if this looks reasonable.

The full output is:

{'position_loss': 83.94,
'rotate6D_loss': 3.25,
'gripper_loss': 0.31,
'loss': 87.50}

Additionally, when evaluating the model on the Libero-10 benchmark, I get a 0% success rate.

This is my script for evaluation:
```Bash
for task_id in 0 1 2 3 4 5 6 7 8 9; do
    echo "Running task $task_id with seed $seed"
    model_path="lerobot/xvla-libero"

    echo "model_path: $model_path"
    lerobot-eval --policy.path=$model_path --policy.n_action_steps=5 \
      --env.type=libero --env.task=libero_10 \
      --env.task_ids="[$task_id]" \
      --eval.n_episodes=50 --eval.batch_size=10 \
      --seed=${seed} \
      --output_dir="outputs/eval/2026-04-17-libero/task${task_id}"
done
```

Could you please advise what might be going wrong or how I can fix this?

Thanks!

### Context & Reproduction
```Python
import torch
from lerobot.datasets.lerobot_dataset import LeRobotDataset
from lerobot.policies.factory import make_pre_post_processors

# Swap this import per-policy
from lerobot.policies.xvla.modeling_xvla import XVLAPolicy

# load a policy
model_id = "lerobot/xvla-libero"  # <- swap checkpoint
device = torch.device("cuda" if torch.cuda.is_available() else "cpu")

policy = XVLAPolicy.from_pretrained(model_id).to(device).eval()

preprocess, postprocess = make_pre_post_processors(
    policy.config,
    model_id,
    preprocessor_overrides={"device_processor": {"device": str(device)}},
)
# load a lerobotdataset (we will replace with a simpler dataset)
dataset = LeRobotDataset("lerobot/libero")

# pick an episode
episode_index = 0

# each episode corresponds to a contiguous range of frame indices
from_idx = dataset.meta.episodes["dataset_from_index"][episode_index]
to_idx   = dataset.meta.episodes["dataset_to_index"][episode_index]

# get a single frame from that episode (e.g. the first frame)
frame_index = from_idx
frame = dict(dataset[frame_index])

batch = preprocess(frame)
with torch.inference_mode():
    pred_action = policy.select_action(batch)
    # use your policy postprocess, this post process the action
    # for instance unnormalize the actions, detokenize it etc..
    pred_action = postprocess(pred_action)

policy.train()
batch = dict(dataset[0])
batch = preprocess(batch)

loss, outputs = policy.forward(batch)
print("Loss:", loss)
print(outputs)
```

```Bash
for task_id in 0 1 2 3 4 5 6 7 8 9; do
    echo "Running task $task_id with seed $seed"
    model_path="lerobot/xvla-libero"

    echo "model_path: $model_path"
    lerobot-eval --policy.path=$model_path --policy.n_action_steps=5 \
      --env.type=libero --env.task=libero_10 \
      --env.task_ids="[$task_id]" \
      --eval.n_episodes=50 --eval.batch_size=10 \
      --seed=${seed} \
      --output_dir="outputs/eval/2026-04-17-libero/task${task_id}"
done
```

### Relevant logs or stack trace

```Shell

```

### Checklist

- [x] I have searched existing tickets to ensure this isn't a duplicate.
- [x] I am using the latest version of the `main` branch.
- [x] I have verified this is not an environment-specific problem.

**Comment by yangmingbj** [comment]
same issue

**Comment by pkooij** [comment]
Do you maybe have one of the rollout video's?

**Comment by Hoar012** [comment]
https://github.com/user-attachments/assets/e65c8430-a409-470a-b23e-0ccdecb70591
Sure, it seems there are only some meaningless actions.

**Comment by Hoar012** [comment]
I've found the solution. I need to set `--env.control_mode=absolute` because X-VLA is trained with absolute poses. Thank you!
